src/cache/DbQueryResultCache.ts

```diff
@@ -29,7 +29,7 @@ export class DbQueryResultCache implements QueryResultCache {
 
         const options = <SqlServerConnectionOptions|PostgresConnectionOptions>this.connection.driver.options;
         const cacheOptions = typeof this.connection.options.cache === "object" ? this.connection.options.cache : {};
-        const cacheTableName = cacheOptions.tableName || "query-result-cache";
+        const cacheTableName = cacheOptions.tableName || "queryresultcache";
```

Comment:
Why this change?

Reply:
Oracle doesn't like table names with dashes inside it as it will then throw an ORA exception. As per other comments, this is an early draft as I came across issues but definitely not the final solution